.github/workflows/notify-plugin-tools.yml

```diff
@@ -14,26 +17,29 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
+          persist-credentials: false
 
       - name: Check for relevant file changes
         id: check_files
         run: |
-          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.merge_commit_sha }})
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} $COMMIT_SHA)
           echo "$CHANGED_FILES" > /tmp/changed_files.txt
           if grep -i -E "\.(ts|tsx|go)$" /tmp/changed_files.txt; then
             echo "has_relevant_changes=true" >> "$GITHUB_OUTPUT"
           else
             echo "has_relevant_changes=false" >> "$GITHUB_OUTPUT"
           fi
         shell: bash
+        env:
+          COMMIT_SHA: ${{ github.event.pull_request.merge_commit_sha }}
 
       - name: Generate token
         if: steps.check_files.outputs.has_relevant_changes == 'true'
         id: generate_token
-        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+        uses: actions/create-github-app-token@3ff1caaa28b64c9cc276ce0a02e2ff584f3900c5 # v2.0.2
         with:
-          app_id: ${{ secrets.APP_ID }}
-          private_key: ${{ secrets.APP_PEM }}
+          app-id: ${{ secrets.APP_ID }}
```

Comment:
Let's change this to use vault. 

secrets for this repo in

plugins-platform-bot-app:app_id
plugins-platform-bot-app:app_pem

Reply:
Added in https://github.com/grafana/grafana-plugin-examples/pull/510/commits/2d6b17a02954daf618b4b7294f8daeba0b55bec4